source/auth/auth.rst

```diff
@@ -179,6 +179,18 @@ specified and has not been negotiated for that credential:
   instructions for when the ``saslSupportedMechs`` field is not present in
   an ``isMaster`` response.
 
+Caching credentials in SCRAM
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In the implementation of SCRAM authentication mechanisms (e.g. SCRAM-SHA-1
+and SCRAM-SHA-256), drivers MUST maintain a cache of computed SCRAM credentials.
+The cache entries SHOULD be identified by some combination of the password, 
```

Comment:
Quoting @xdg in https://github.com/mongodb/specifications/pull/297#discussion_r177281363:

> but be sure to distinguish what passwords are relevant -- i.e. the mongodb hashed one (SHA-1) or the SASLprepped one (SHA-2).

This may call for a sentence at the end of this paragraph to note "hashed password" for SHA-1, etc.